store/cassandra/cassandra.go

```diff
@@ -219,28 +222,42 @@ func NewCassandraStore(config *StoreConfig, ttls []uint32) (*CassandraStore, err
 	if err != nil {
 		return nil, err
 	}
-	log.Debugf("CS: created session with config %+v", config)
+
+	sd := make(chan struct{})
+
+	cs := cassandra.NewSession(session, cluster, sd, config.ConnectionCheckTimeout, config.ConnectionCheckInterval, config.Addrs, "cassandra_store")
```

Comment:
it's a bit inconsistent with other log statements to call it `cassandra_store` instead of `cassandra-store`. in most other cases (including cassandra idx) we use `-` as separator

Reply:
I completely agree. I was just leaving it as I found it. I would not mind changing it though.